sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -190,6 +191,8 @@ public async Task DeleteSettingWithLabel()
                 });
 
                 Assert.AreEqual(404, e.Status);
+                Assert.IsNotEmpty(e.Message);
+                Assert.IsFalse(e.Message.Contains(troubleshootingLink));
```

Comment:
This new assertion is not related to what this test is validating (deletion of settings with labels). I believe the mock test you created should be enough to validate this behavior. If you feel strongly about having a live test, I recommend creating a new one.

Reply:
I've removed the changes from the live test and only added a mock test.